tasks/insights-client.yml

```diff
@@ -36,6 +36,32 @@
     insertafter: "#auto_update"
     line: auto_update={{ rhc_insights.autoupdate | d(true) | bool }}
 
+- name: Configure ansible host
```

Comment:
Not sure why you are using a `block` here - did you mean to use the `when` condition for the tasks in the block?

```yaml
  when: rhc_insights.ansible_host | d() != None
  block:
....
```
then you don't need the `when` for `ansible_host` below

Reply:
I made the block and forgot to pull out the `when` condition.
Thanks for the suggestion, it simplifies the tasks in the block a bit.